utils/netbox_major_version

```diff
@@ -1 +1 @@
-v3.3
```

Comment:
Hello,

Thanks for this PR.
I prefer to not change the version of Netbox in this PR.

Reply:
Fixed.